sdk/core/Azure.Core/src/Pipeline/Internal/RequestActivityPolicy.cs

```diff
@@ -18,6 +18,7 @@ internal class RequestActivityPolicy : HttpPipelinePolicy
         private const string RequestIdHeaderName = "Request-Id";
 
         private static readonly DiagnosticListener s_diagnosticSource = new DiagnosticListener("Azure.Core");
+        private static readonly object? s_activitySource = ActivityExtensions.CreateActivitySource("Azure.Core.Http");
```

Comment:
Not a breaking change because we are still going to send Activities through the old DiagnosticListener. ActivitySource and DiagnosticListener have a bit different naming guidelines. We used the namespace as the DiagnosticListener name but use the type name or the area name for ActivitySource.